notifications/02-01-23-1100.md

```diff
@@ -0,0 +1,97 @@
+**What is happening?**
+
+- **GOLD** 2023 Q1 Patching
+
+  - CHG0044925 - Manual Image Prune, ETCD Defrag, iSCSI LIF addition and Trident Upgrade 
+
+  - CHG0044930 - Upgrade to OCP 4.11.18
+
+These two changes represent the work involved to upgrade the production **GOLD** cluster from Openshift 4.10 to 4.11.
+
+- **GOLDDR** 2023 Q1 Patching
+
+  - CHG0044932 -  Manual Image Prune, ETCD Defrag, iSCSI LIF addition and Trident Upgrade
+
+  - CHG0044931 - Upgrade to OCP 4.11.18
+
+These two changes represent the work involved to upgrade the production **GOLDDR** cluster from Openshift 4.10 to 4.11.
+
+**When?**
+
+- **GOLD** 2023 Q1 Patching
+
+  - CHG0044925 - 6:00am on Sunday February 5th until until 10:00 a.m. the same day.
+
+  - CHG0044930 - 9:00 am on Monday, February 6th to 5:00 pm on Friday, February 10th. Upgrade efforts will be allowed to proceed between 9:00am and 5:00pm, and be paused after-hours to ensure developers are present during node drains to look after their apps if needed.
+
+- **GOLDDR** 2023 Q1 Patching
+
+  - CHG0044932 - 6:00 am on Sunday February 12th until until 10:00 a.m. the same day.
+
+  - CHG0044931 - 9:00 am on Monday, February 13th to 5:00 pm on Friday, February 17th. Upgrade efforts will be allowed to proceed between 9:00am and 5:00pm, and be paused after-hours to ensure developers are present during node drains to look after their apps if needed.
+
+**Will there be an impact on the Platform apps?**
+
+CHG0044925(GOLD) and CHG0044932(GOLDDR) will involve removing some old images and defragmenting the ETCD database. Image tags that are not currently in use by a pod or deployment and are older than 14 days will be pruned from the registry completely. Defragmenting ETCD could cause some latency and slowness but isn't expected to cause any breakage as each pod will be done individually with time to recover between operations. The iSCSI LIF addition should be entirely transparent to users. The Trident upgrade may cause a brief window where new storage cannot be allocated, expected to be under 5 minutes. It will not impact existing storage.
+
+CHG0044930(GOLD) and CHG0044931(GOLDDR) will involve both an upgrade of the core of the cluster as well as draining/reboots of all nodes which will require pods to be drained onto other nodes. There may be some slowness for new pods coming up if it happens that a lot of new pods attempt to start on an empty node (just patched). We will announce daily when work for this will resume and be suspended at the end of each work day. Depending on availability of new firmware, there may be a need for us to do another node drain after Openshift upgrade for firmware.
```

Comment:
```suggestion
CHG0044930(GOLD) and CHG0044931(GOLDDR) will involve both an upgrade of the core of the Openshift cluster as well as draining/reboots of all nodes which will require pods to be drained onto other nodes. There may be some slowness for new pods coming up if it happens that a lot of new pods attempt to start on an empty node (just patched). We will announce daily when work for this will resume and be suspended at the end of each work day. Depending on availability of new firmware, there may be a need for us to do another node drain after Openshift upgrade for firmware.
```